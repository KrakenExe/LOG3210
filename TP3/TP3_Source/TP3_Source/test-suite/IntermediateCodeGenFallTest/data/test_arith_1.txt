Declaration
num a;
num b;
num c;
EndDeclaration

a = 1;
b = 2;
c = 3;