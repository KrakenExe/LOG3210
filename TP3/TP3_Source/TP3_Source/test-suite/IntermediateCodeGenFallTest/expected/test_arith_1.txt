a = 1
_L1
b = 2
_L2
c = 3
_L0
